toml.abnf

```diff
@@ -84,9 +84,10 @@ ml-basic-string = ml-basic-string-delim ml-basic-body ml-basic-string-delim
 
 ml-basic-string-delim = 3quotation-mark
 
-ml-basic-body = *( ml-basic-char / newline / ( escape ws newline ) )
-ml-basic-char = ml-basic-unescaped / escaped
+ml-basic-body = *( ml-basic-char / wschar / newline / ml-basic-escaped-nl )
```

Comment:
The `wschar` token seems unnecessary here, since it's already part of `ml-basic-char`.

Reply:
Indeed, this was also not present in the original input.